stories/instantsearch.stories.js

```diff
@@ -1,17 +1,31 @@
 import { storiesOf } from '@storybook/html';
 import { withHits } from '../.storybook/decorators';
 
-storiesOf('InstantSearch', module).add(
-  'with searchfunction to prevent search',
-  withHits(() => {}, {
-    searchFunction: helper => {
-      const query = helper.state.query;
+storiesOf('InstantSearch', module)
+  .add(
+    'with searchfunction to prevent search',
+    withHits(() => {}, {
+      searchFunction: helper => {
+        const query = helper.state.query;
 
-      if (query === '') {
-        return;
-      }
+        if (query === '') {
+          return;
+        }
 
-      helper.search();
-    },
-  })
-);
+        helper.search();
+      },
+    })
+  )
+  .add(
+    'with initialUiState',
+    withHits(() => {}, {
+      initialUiState: {
+        // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
was thinking on allowing the string `instant_search` specifically, or else we can do it with indexId, but also fine to just ignore here

Reply:
Not sure it's really useful, until now we don't have it that much inside the codebase.